src/controllers/clientController.js

```diff
@@ -128,7 +128,7 @@ exports.getOrderHistory = async (req, res) => {
     const decodedToken = decodeJwtToken(token);
     const client = await Client.findOne({
       userId: decodedToken.userId,
-    }).populate('orderHistory');
+    }); // .populate('orderHistory');
```

Comment:
I think we should get rid of the test that needs to delete the populate, can you get it back please?